src/lib/support/CHIPArgParser.cpp

```diff
@@ -911,9 +911,9 @@ bool ParseInt(const char * str, int16_t & output)
     const int base   = 10;
     int32_t output32 = 0;
 
-    if ((ParseInt(str, output32, base)) && (output32 <= SHRT_MAX))
+    if ((ParseInt(str, output32, base)) && (output32 <= SHRT_MAX && output32 >= SHRT_MIN))
```

Comment:
Preexisting issue but: it's INT32_MIN/MAX

Reply:
The existing check is correct because `output32` will be converted to `int16_t` below.

Reply:
Suggest using INT16_MIN / INT16_MAX though.